packages/esm-patient-common-lib/src/lab-resource/types/order.ts

```diff
@@ -0,0 +1,16 @@
+import { type Order } from '@openmrs/esm-patient-common-lib';
+
+export interface PatientMedicationFetchResponse {
```

Comment:
Why is _this_ type in `/lab-resource/types`. It seems like it definitely belongs somewhere else.